src/chains/subsocial/api/events.ts

```diff
@@ -1,7 +1,8 @@
 import { ChainContext, Event } from '../types/support';
 import {
   AccountFollowsAccountFollowedEvent,
-  AccountFollowsAccountUnfollowedEvent,
+  AccountFollowsAccountUnfollowedEvent, DomainsDomainMetaUpdatedEvent,
```

Comment:
isn't it better to add newline to this import

Reply:
Fixed